### ISSUE TITLE
About the Reproducibility of the paper

### ISSUE THREAD

**Issue by FayeXXX** [Tikquuss/controllable_text_attribute_transfer#1]
Thank you for your code, and did you get the same result reported on the original paper?

**Comment by Tikquuss** [comment]
No, the authors' code does not reflect the algorithms presented in the paper.

Only the authors know how they got such results.
